drivers/input/Kconfig.cst816s

```diff
@@ -25,4 +25,10 @@ config INPUT_CST816S_INTERRUPT
 	help
 	  Enable interrupt support (requires GPIO).
 
+config INPUT_CST816S_EV_DEVICE
+	bool "Device specific event support"
+	default n
```

Comment:
For bool this is not needed

Reply:
@fabiobaltieri , @kartben , is this a blocker?